extract_unique_sensors_01.py

```diff
@@ -13,45 +17,58 @@ def create_parser():
 
     parser = argparse.ArgumentParser(description='Extract unique sensors')
     parser.add_argument('--file_name', help='File to extract unique sensors from.')
-    parser.add_argument('--url', help='A reference to SensorList.txt that specifies its ' 
-                        'location on a computer network.') 
+parser.add_argument(
+    '--url',
+    help='A reference to SensorList.txt that specifies its location on a computer network.'
+)
```

Comment:
I broke your code, forgot the indentation. Sorry! :wink: 
```suggestion
    parser.add_argument(
        '--url',
        help='A reference to SensorList.txt that specifies its location on a computer network.'
    )
```

Reply:
This implementation breaks the 90 characters per line rule.( 95 > 90 characters)

Reply:
https://www.softwaretestingmaterial.com/python-multiline-string/